cmd/server.go

```diff
@@ -81,6 +82,7 @@ func serverRun(cmd *cobra.Command, args []string) {
 	}
 
 	go func() {
+		var oldCheckMetrics map[int]prometheus.Labels = nil
```

Comment:
By default, a new uninitialized variable is _zero-valued_. So, in your case, it would be better to write:
```golang
var oldCheckMetrics map[int]prometheus.Labels
```

Reply:
Thanks, I modified it.